packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -509,7 +509,13 @@ export const TooltipPlugin2 = ({
 
   if (plot && isHovering) {
     return createPortal(
-      <div className={cx(styles.tooltipWrapper, isPinned && styles.pinned)} style={style} ref={domRef}>
+      <div
+        className={cx(styles.tooltipWrapper, isPinned && styles.pinned)}
+        style={style}
+        aria-live="polite"
```

Comment:
Are these aria labels common for any dynamically updated content in a component? Haven't seen it used before, but if it's common for a11y, I wanna make a note to myself. 🤔

Reply:
not sure, check `VizTooltipContainer` commit history for context. i just copy/paste :)